src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -506,7 +506,7 @@ protected override void ProcessRecord()
         {
             if (EffectiveParameterSet == MyParameterSet.RandomListItem)
             {
-                if (ParameterSetName == ShuffleSet)
+                if (ParameterSetName == ShuffleParameterSet)
```

Comment:
I'm not sure about this condition. Should we compare strings to check the `Shuffle` parameter is specified, or it would be better to check just the `Shuffle.IsPresent` property?

Reply:
I think checking either should be sufficient, really, but for consistency's sake we should probably keep it to checking the parameter name.